.prettierrc

```diff
@@ -1,5 +1,6 @@
 {
   "semi": true,
   "trailingComma": "all",
-  "singleQuote": true
+  "singleQuote": true,
+  "arrowParens": "avoid"
```

Comment:
This isn't specific to this PR, but I didn't want to make a whole 'nother branch just for this :upside_down_face: 

https://prettier.io/docs/en/options.html#arrow-function-parentheses